packages/auth0/src/error-handling-middleware.ts

```diff
@@ -0,0 +1,33 @@
+import type { Request, Response, NextFunction } from 'express';
+
+export function defaultErrorHandler(error: Error, _req: Request, res: Response, next: NextFunction) {
+  if (res.headersSent) {
+    return next(error);
+  }
+
+  let assertCondition = 'Assert condition failed: ';
+
+  if (error?.message?.startsWith(assertCondition)) {
+    let errorCode = 500;
+    let errorResponse = error.message;
+
+    if (error.message.includes('::')) {
+      let errorMessage = error.message.slice(assertCondition.length);
+      errorCode = parseInt(errorMessage.slice(0, 3));
+      errorResponse = errorMessage.slice(5);
+    }
+
+    res.status(errorCode).send(errorResponse);
+  } else {
+    console.error(error);
```

Comment:
I accidentally forgot to remove this, but honestly.... I kind of don't mind it. It is nice to log out these unhandled conditions loudly rather than just silently failing with this stuff.